website/integrations/services/actual-budget/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Integrate with Actual Budget
+sidebar_label: Actual Budget
+---
+
+# Actual Budget
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Actual Budget
+
+> Actual Budget is a super-fast and privacy-focused app for managing your finances. At its core is the well-proven and much-loved envelope budgeting methodology.
+> You own your data and can do whatever you want with it. It features multi-device sync, optional end-to-end encryption, and much more.
+>
+> -- https://actualbudget.org/
+>
+> This guide explains how to configure Actual Budget to use authentik as the OAuth provider for logging in to the Web GUI.
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `actual.company` is the FQDN of the Actual Budget install.
+- `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+[Create](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#add-new-applications) an OAuth2/OpenID provider and an application in authentik. Use the following parameters for the OAuth2/OpenID provider:
+
+**Provider:**
+
+- Name: `SP-actual`
+- Client type: `Confidential`
+- Redirect URIs/Origins (RegEx): `https://actual.company/openid/callback`
+- Signing Key: Select any available signing keys.
+
+:::info
+Actual Budget supports the RS256 algorithm. Please ensure to consider this when choosing the appropriate signing key.
+:::
+
+Take note of the Client ID and Client Secret; you will need to provide them to Actual Budget in Step 3.
+
+Leave the remaining values as default. Durations can be adjusted as needed.
+
+**Application:**
+
+- Name: `Actual Budget`
+- Slug: `actual`
+- Launch URL: `https://actual.company/`
+
+## Actual Budget configuration
+
+1. Sign in to Actual Budget with a browser of your choice and access your budget by clicking on its name.
+
+2. Click your budget in the top-left corner to open the dropdown menu and select `Settings`.
+
+3. Scroll to the bottom and select `Show advanced settings`. Scroll again and select `I understand the risks, show experimental features`.
+
+4. Enable the option `OpenID authentication method` by selecting the checkbox next to it.
+
+5. Scroll up to the new option `Authentication method...` and click the `Start using OpenID` button.
+
+6. Enter the variables provided by authentik here.
+    - Set `OpenID Provider` to `Authentik`
+    - Set `OpenID provider URL` to `https://authentik.company/application/o/actual/`
+    - Set `Client ID` to `<client-id>` from Step 2
```

Comment:
```suggestion
    - Set `Client ID` to `<client-id>`
```

Reply:
Do you mean Step 2.. under the authentik configuration section (which is not numbered with steps) or ...?

Reply:
Yes but your suggestion is absolutely more understandable 💪